src/COMDIRAC/Interfaces/scripts/dkill.py

```diff
@@ -79,7 +79,7 @@ def main():
         if result["OK"]:
             userName = result["Value"]
 
-            monitoring = RPCClient("WorkloadManagement/JobMonitoring")
+            monitoring = Client(url="WorkloadManagement/JobMonitoring")
```

Comment:
it can be ``from DIRAC.WorkloadManagementSystem.Client.JobMonitoringClient import JobMonitoringClient`` I think, no?